bedrock/base/templates/macros.html

```diff
@@ -591,6 +599,7 @@
             <li><a href="{{ url('firefox.os.devices') }}" data-id="os-devices"><div>{{ _('Devices') }}</div></a></li>
             <li><a href="{{ url('firefox.partners.index') }}" data-id="os-partners"><div>{{ _('For partners') }}</div></a></li>
             <li><a rel="external" href="https://marketplace.firefox.com/" data-id="os-marketplace"><div>{{ _('Marketplace') }}<i class="icon-external"></i></div></a></li>
+            <li><a rel="external" href="https://marketplace.firefox.com/" data-id="os-apps"><div>{{ _('Apps') }}<i class="icon-external"></i></div></a></li>
```

Comment:
The navigation system is in a shared file (main.lang), so we simply need to hope that it will be localized in time (had the same issue with the other strings).